private/buf/bufcli/bufcli.go

```diff
@@ -802,26 +802,26 @@ func NewImageForSource(
 	return bufimage.MergeImages(images...)
 }
 
-// ParseSourceAndType returns the moduleReference and typeName from the source and type provided by the user.
-// When source is not provided, we assume the type is a fully qualified path to the type and try to parse it.
-// Otherwise, if both source and type are provided, the type must be a valid Protobuf identifier (e.g. weather.v1.Units).
-func ParseSourceAndType(
+// ParseInputAndType returns the moduleReference and typeName from the input and type provided by the user.
+// When input is not provided, we assume the type is a fully qualified path to the type and try to parse it.
+// Otherwise, if both input and type are provided, the type must be a valid Protobuf identifier (e.g. weather.v1.Units).
+func ParseInputAndType(
```

Comment:
I am actually not really sure on should we rename everything to Input, for example the source and type here seems reflecting the relationship of fields better? not sure though

Reply:
Reverted these changes

Reply:
I think that it's closer to what the docs say when it's "input" as it could be either source or image at this point so imo image makes sense